test/minikube/minikube_tls_admin_test.go

```diff
@@ -45,7 +45,7 @@ func TestKubernetesTLS(t *testing.T) {
 	defer k8s.DeleteNamespace(t, kubectlOptions, namespaceName)
 
 	defer testlib.Teardown(testlib.TEARDOWN_SECRETS)
-	
+
```

Comment:
did you somehow configure an IDE to clean these up automatically?

Reply:
It happens automatically with Spacemacs (Emacs distribution) with the go layer.